libraries/report.rb

```diff
@@ -61,8 +55,6 @@ class ComplianceReport < Chef::Resource
           else
             Chef::Log.warn "#{collector} is not a supported inspec report collector"
           end
-
-          raise "#{total_failed} audits have failed.  Aborting chef-client run." if total_failed > 0 && node['audit']['fail_if_any_audits_failed']
```

Comment:
this is just a quick hack, but _if this functionality should be retained_, we could go with something like

```ruby
total_failed = reports.map do |name, report|
  report['profiles'].map do |profile|
    profile['controls'].map do |control|
      control['results'].map do |result|
        result['status'] != 'passed' ? 1 : 0
      end
    end
  end
end.flatten.reduce(:+)
```